tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py

```diff
@@ -158,7 +158,6 @@ def test_ingest_consumer_fails_when_not_autocreating_topics(kafka_admin, random_
     assert kafka_error.code() == KafkaError.UNKNOWN_TOPIC_OR_PART
 
 
-@pytest.mark.xfail(reason="fixme")
```

Comment:
This test is now fixed and therefore enabled. This was the initial reason for this PR.